specification/dsdl/serialization.tex

```diff
@@ -426,16 +426,78 @@ \subsection{Composite types}
-        \underbrace{\hugett{11}}_{\texttt{fourth}}%
-        \overbrace{\hugett{100,0}}^{\texttt{fifth}}%
-        \underbrace{\hugett{???????}}_{\substack{\text{Next object or} \\ \text{zero padding bits}}}
+        \underbrace{%
+            \overbrace{%
+                \underset{7}{\overset{7}{\hugett{1}}}%
+                \underset{6}{\overset{6}{\hugett{1}}}%
+                \underset{5}{\overset{5}{\hugett{0}}}%
+                \underset{4}{\overset{4}{\hugett{1}}}%
+                \underset{3}{\overset{3}{\hugett{1}}}%
+                \underset{2}{\overset{2}{\hugett{0}}}%
+                \underset{1}{\overset{1}{\hugett{1}}}%
+                \underset{0}{\overset{0}{\hugett{0}}}%
+            }^{\texttt{first}}%
+        }_{\texttt{byte 0}}%
+        \hugett{,}%
+        \underbrace{%
+            \overbrace{%
+                \underset{7}{\overset{0}{\hugett{1}}}%
+            }^{\texttt{third}}%
+            \overbrace{%
+                \underset{6}{\overset{2}{\hugett{1}}}%
+                \underset{5}{\overset{1}{\hugett{1}}}%
+                \underset{4}{\overset{0}{\hugett{1}}}%
+            }^{\texttt{second}}%
+            \overbrace{%
+                \underset{3}{\overset{11}{\hugett{1}}}%
+                \underset{2}{\overset{10}{\hugett{1}}}%
+                \underset{1}{\overset{9}{\hugett{0}}}%
+                \underset{0}{\overset{8}{\hugett{1}}}%
+            }^{\texttt{first}}%
+        }_{\texttt{byte 1}}%
+        \hugett{,}%
+        \underbrace{%
+            \overbrace{%
+                \underset{7}{\overset{2}{\hugett{0}}}%
+                \underset{6}{\overset{1}{\hugett{0}}}%
+                \underset{5}{\overset{0}{\hugett{0}}}%
+            }^{\texttt{fifth}}%
+            \overbrace{%
+                \underset{4}{\overset{1}{\hugett{1}}}%
+                \underset{3}{\overset{0}{\hugett{1}}}%
+            }^{\texttt{fourth}}%
+            \overbrace{%
+                \underset{2}{\overset{3}{\hugett{1}}}%
+                \underset{1}{\overset{2}{\hugett{0}}}%
+                \underset{0}{\overset{1}{\hugett{1}}}%
+            }^{\texttt{third}}%
+        }_{\texttt{byte 2}}%
+        \hugett{,}%
+        \underbrace{%
+            \overbrace{%
+                \underset{7}{\overset{?}{\hugett{?}}}%
+                \underset{6}{\overset{?}{\hugett{?}}}%
+                \underset{5}{\overset{?}{\hugett{?}}}%
+                \underset{4}{\overset{?}{\hugett{?}}}%
+                \underset{3}{\overset{?}{\hugett{?}}}%
+                \underset{2}{\overset{?}{\hugett{?}}}%
+                \underset{1}{\overset{?}{\hugett{?}}}%
+            }^{\substack{\text{Next object or} \\ \text{zero padding bits}}}
+            \overbrace{%
+                \underset{0}{\overset{3}{\hugett{1}}}%
+            }^{\texttt{fifth}}%
+        }_{\texttt{byte 3}}%
     $$
+
+    Note that some of the complexity of the above illustration stems from the western convention of representing
```

Comment:
> western convention

Hah!

Why Western though? Wikipedia says that the first [positional numeral system](https://en.wikipedia.org/wiki/Positional_notation) (presumably left-to-right) was [invented in Babylon](https://en.wikipedia.org/wiki/Babylonian_cuneiform_numerals); the numeral system we use today across the world ([Western Arabic](https://en.wikipedia.org/wiki/Arabic_numerals)) was invented when Europe was knee-deep in the Dark Ages, way before the modern concept of The West. We probably shouldn't venture to historical matters too much so I suggest we just say "modern".

```suggestion
    Note that some of the complexity of the above illustration stems from the modern convention of representing
```

Reply:
Okay. I was just guessing about the history.